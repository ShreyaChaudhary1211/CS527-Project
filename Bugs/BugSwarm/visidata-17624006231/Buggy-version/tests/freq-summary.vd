sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Quantity		type-int		#	
benchmark			select-expr	Quantity > 4	z|	
benchmark	Quantity		aggregate-col	mean	+	
benchmark			freq-summary		zF	
