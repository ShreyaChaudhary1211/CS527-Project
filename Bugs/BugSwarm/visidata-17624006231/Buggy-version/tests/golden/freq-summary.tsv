Total	count	Quantity_mean
Total	51	7.90
Selected	11	29.36
